src/main/charts/confluence/values.yaml

```diff
@@ -183,6 +183,18 @@ synchrony:
     periodSeconds: 1
     # -- The number of consecutive failures of the Synchrony container readiness probe before the pod fails readiness checks
     failureThreshold: 30
+  resources:
```

Comment:
Might be nice to update the test `[output.yaml](https://github.com/atlassian-labs/data-center-helm-charts/blob/master/src/test/resources/expected_helm_output/confluence/output.yaml)`  for confluence to include these values so that their presence and corresponding values are tested in future

Reply:
The `HelmOutputComparisonTest` is verifying an actual output of the processed `helm template` with default values and compare it to the recorded response. If the default is not changed, then the test doesn't see any difference to compare.

The main problem is that the default is `synchrony.enabled: false`. I think we have two options.

1. Enable synchrony by default, this seems like a better option for customers as they get more functionality out of the box. The risk is that there might be some misconfiguration why it doesn't work. @badgersow, what do you think, does it make sense to enable synchrony by default?
2. Add `synchrony.enabled: true` to https://github.com/atlassian-labs/data-center-helm-charts/blob/master/src/test/resources/expected_helm_output/confluence/values.yaml and that will enable it at least for the comparison test.

Either way I think we should test for synchrony output as it is fairly chunky part of the solution that is not covered much (we have `SynchronyTest` unit test).

Reply:
Yup - What I was getting at, we should test for synchrony output

Reply:
I've add a test for the entrypoint output

Reply:
We should definitely enable synchrony by default. The next `7.12.2` version of Confluence should have all changes for Collab editing to be also enabled by default. Confluence builds test the case when Synchrony is enabled.

Reply:
Should I do this as part of this PR, so should we treat that separately?

Reply:
We can do it as part of this PR but I can see you've already done it. 🙇‍♂️